src/Illuminate/Queue/Worker.php

```diff
@@ -195,7 +197,9 @@ public function process($connection, Job $job, $maxTries = 0, $delay = 0)
 	{
 		if ($maxTries > 0 && $job->attempts() > $maxTries)
 		{
-			return $this->logFailedJob($connection, $job);
+			$exception = new RuntimeException(sprintf('Max tries of %d attempted.', $maxTries));
+
```

Comment:
I'm a little confused by this check to. How would it ever be hit? If a job didn't throw an exception but was released to many times?